### DISCUSSION TITLE
Movable egg variables

### DISCUSSION THREAD

**Discussion by HerrSammyDE** [pelican-dev/panel#62] [Ideas]
Currently the egg variables are all created one after the other, but I would very much like to be able to sort / move the variables, so a function to move them would be very good and helpful

**Comment by HerrSammyDE** [comment]
maybe like https://filamentphp.com/docs/3.x/forms/fields/repeater#grid-layout

**Comment by lancepioch** [comment]
Follow the issue here: https://github.com/pelican-dev/panel/issues/72
